dockstore-integration-testing/src/test/resources/fixtures/createDepositResponse.json

```diff
@@ -0,0 +1,34 @@
+{
+  "conceptrecid": {{ randomIntegerRange 10000 99999 }},
```

Comment:
Neat (and the `Request.Path[3]` further as well)! How did I not know about this? Where is the documented? I couldn't find it in a quick google search.

Reply:
It's documented here :slightly_smiling_face:  https://docs.hoverfly.io/en/latest/pages/keyconcepts/templating/templating.html